.github/workflows/build.yaml

```diff
@@ -29,6 +29,8 @@ jobs:
     - name: Dependency Review
       uses: actions/dependency-review-action@v4
       with:
+        base-ref: ${{ github.event.pull_request.base.sha || '0.1.0' }}
```

Comment:
In https://github.com/actions/dependency-review-action it says that `base-ref/head-ref` is used for event types other than `pull_request` and `pull_request_target`. Why do we need the `github.event.pull_request.base.sha` and `github.event.pull_request.head.sha` then?

Reply:
as far as I can see their documentation is imprecise, see https://github.com/actions/dependency-review-action/blob/935098a9508277d66b6c3adc56de47383d4c1e56/src/git-refs.ts#L10-L11

Reply:
Interesting, so if we define `base-ref` and `head-ref` they'll override PRs events as well.